SpatialGDK/Source/SpatialGDKEditor/Public/SpatialGDKEditorSettings.h

```diff
@@ -7,6 +7,187 @@
 
 #include "SpatialGDKEditorSettings.generated.h"
 
+USTRUCT()
+struct FWorldLaunchSection
+{
+	GENERATED_BODY()
+
+	FWorldLaunchSection()
+		: Dimensions(2000, 2000)
+		, ChunkEdgeLengthMeters(50)
+		, StreamingQueryIntervalSeconds(4)
+		, SnapshotWritePeriodSeconds(0)
+	{
+		LegacyFlags.Add(TEXT("bridge_qos_max_timeout"), TEXT("0"));
+		LegacyFlags.Add(TEXT("bridge_soft_handover_enabled"), TEXT("false"));
+		LegacyFlags.Add(TEXT("enable_chunk_interest"), TEXT("false"));
```

Comment:
@m-samiec @BenNaccarato This is already disabled, is there anything else needed before I can merge this?

Reply:
Looks good.